cmd/mc/skin/add.go

```diff
@@ -0,0 +1,125 @@
+package skin
+
+import (
+	"errors"
+	"fmt"
+	"slices"
+
+	"github.com/google/uuid"
+	"github.com/mworzala/mc/internal/pkg/cli"
+	"github.com/spf13/cobra"
+)
+
+type addSkinOpts struct {
+	app *cli.App
+
+	account string
+	variant string
+	cape    string
+	name    string
+	apply   bool
+}
+
+var (
+	ErrInvalidType    = errors.New("invalid type")
+	ErrInvalidVariant = errors.New("invalid variant")
+
+	validVariants = []string{"classic", "slim"}
```

Comment:
this is also a mega nitpick but the more typical pattern here is a map[string]bool then you can just do `if !validationMap[variant] { bad variant }`